src/Psalm/Internal/Fork/PsalmRestarter.php

```diff
@@ -181,4 +193,23 @@ protected function restart($command): void
 
         parent::restart($command);
     }
+
+    /**
+     * @return positive-int
+     */
+    private function getRequiredMemoryConsumption(): int
+    {
+        // Reserve for byte-codes
+        $result = 256;
+
+        if (isset(self::REQUIRED_OPCACHE_SETTINGS['jit_buffer_size'])) {
+            $result += self::REQUIRED_OPCACHE_SETTINGS['jit_buffer_size'] / 1024 / 1024;
+        }
+
+        if (isset(self::REQUIRED_OPCACHE_SETTINGS['interned_strings_buffer'])) {
+            $result += self::REQUIRED_OPCACHE_SETTINGS['interned_strings_buffer'];
+        }
+
+        return $result;
```

Comment:
return this in bytes

Reply:
According to the documentation, the value for `memory_consumption` is set in megabytes.